### DISCUSSION TITLE
致广大1.59无法登录，1.61无法解析无法下载的朋友

### DISCUSSION THREAD

**Discussion by Rangtian** [leiurayer/downkyi#1081] [General]
可以使用 https://github.com/yaobiao131/downkyicore

----------------------------------------------------------

备选方案：先下载1.61，扫码登录后，用1.59覆盖。
于2024-03-30成功解析并下载凡人修仙传。

**Comment by xsx666** [comment]
请问覆盖怎么操作？

**Reply by Rangtian** [reply]
就是把1.59文件夹里的所有内容，复制粘贴到1.61文件夹中，替换掉1.61

**Reply by KallenSuteki** [reply]
只需要覆盖Config文件夹下的Login文件即可

**Comment by xybleo** [comment]
> 就是把1.59文件夹里的所有内容，复制粘贴到1.61文件夹中，替换掉1.61

成功登录并下载了，感谢。请问可以关闭每次打开1.59时弹出的自动检测更新窗口吗？谢谢
